src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
```

Comment:
first there should be external imports, then internal
so `import './App.scss';` should goes after other it this list